titanic_datasources/README.md

```diff
@@ -127,30 +127,30 @@
 
 
 ## Section 3: Data Generation
-1. Open [data_generation.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/titanic/data_generation.ipynb) notebook for generating predict and groundtruth datasets.
+1. Open [data_generation.ipynb](https://github.com/oneconvergence/dkube-examples/tree/monitoring/titanic_datasources/data_generation.ipynb) notebook for generating predict and groundtruth datasets.
 2. In 1st cell, Update Frequency according to what you set in Modelmonitor. If the **d3qatest** tag was provided when creating a modelmonitor, specify the frequency in minutes. For eg: for 5 minutes, specify it as `5m`. If **d3qatest** tag was not specified on the modelmonitor, specify the frequency in hours. use `5h` for 5 hours interval.
```

Comment:
this is incorrect right? there is no d3qatest now. Also, frequency is in the 1st cell of resources.ipynb and not here

Reply:
yes, fixing